### ISSUE TITLE
Cannot find resource named 'MaterialDesignComboBox'. Resource names are case sensitive

### ISSUE THREAD

**Issue by UnRunDeaD** [spiegelp/MaterialDesignExtensions#28]
Where is this error coming from?

I downloaded your NuGet Packages via download link, the default materialdesign is already installed. 

I copied your App.XAML into mine and into my MainWindow.

And my Title error Error appears

Have you any Idea ?

Thx a lot

**Comment by UnRunDeaD** [comment]
Hi @spiegelp 
I am using 
MaterialDesignThemes v. 2.5.0.1205, downgraded right now to 2.4.1.1101 and still got the Error.
Extension v. is 2.4.0

What version of MDT did you recommend at the moment ?

**Comment by spiegelp** [comment]
Hi @UnRunDeaD 

MaterialDesignThemes version 2.5.0.1205 has a bug with text fields causing problems with the `OversizedNumberSpinner` control. Futhermore, it updated the icons causing the file controls to show wrong icons, because the enum behind was changed. If you use the file controls, please downgrade to version 2.4.1.1101, until I update MaterialDesignExtensions. Otherwise a newer prerelease will work too. 

Concerning the exception: It seems that some styles of MaterialDesignThemes are missing in your app. Please post your App.xaml that I can check your configuration and the referenced resource dictionaries.

**Comment by UnRunDeaD** [comment]
Thats my App.xaml Code


`<Application x:Class="Converter.App"
             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
             xmlns:local="clr-namespace:Converter"
             StartupUri="MainWindow.xaml">
    <Application.Resources>
        <ResourceDictionary>
            <ResourceDictionary.MergedDictionaries>
                <ResourceDictionary Source="pack://application:,,,/MaterialDesignExtensions;component/Themes/MaterialDesignDarkTheme.xaml" />
                <ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/MaterialDesignTheme.Light.xaml" />
                <ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/MaterialDesignTheme.Defaults.xaml" />
                <ResourceDictionary Source="pack://application:,,,/MaterialDesignColors;component/Themes/Recommended/Primary/MaterialDesignColor.Blue.xaml" />
                <ResourceDictionary Source="pack://application:,,,/MaterialDesignColors;component/Themes/Recommended/Accent/MaterialDesignColor.Indigo.xaml" />
            </ResourceDictionary.MergedDictionaries>
        </ResourceDictionary>
    </Application.Resources>
</Application>
`

**Comment by spiegelp** [comment]
@UnRunDeaD Once I had an issue caused by the order of the resources. Could you try to replace the part before the colors with the following code? 
```xaml
<ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/Generic.xaml" />
<ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/MaterialDesignTheme.Defaults.xaml" />
<ResourceDictionary Source="pack://application:,,,/PhiliStudioMaterialDesignLibrary;component/Themes/MaterialDesignLightTheme.xaml" />
```

**Comment by Yesterday17** [comment]
I've also met this problem and i found it's caused by [FileSystemControlTemplates.xaml](https://github.com/spiegelp/MaterialDesignExtensions/blob/master/MaterialDesignExtensions/Themes/FileSystemControlTemplates.xaml).

**Comment by spiegelp** [comment]
@Yesterday17 I will check the situation and try to list explicitly all used resources per XAML file.

**Comment by DenZuck** [comment]
Hi @spiegelp I get this error in app.xaml

My configuration is:
- Visual Studio Community 2017 v. 15.9.11
- MaterialDeisignColors v. 1.1.3
- MaterialDeisignExtensions v. 2.5.0.1
- MaterialDesignThemes v. 2.5.1

I've attached an example

[TestMaterialExtensions.zip](https://github.com/spiegelp/MaterialDesignExtensions/files/3162404/TestMaterialExtensions.zip)

And this is the error

<img width="1316" alt="Schermata 2019-05-09 alle 15 45 57" src="https://user-images.githubusercontent.com/7376037/57458408-95b44100-7271-11e9-8f52-c84adff1cf94.png">

**Comment by Tyrrrz** [comment]
I also get this error when using SaveFileDialog

**Comment by spiegelp** [comment]
Hi everybody!

I uploaded a pre-release (version 2.6.0-a02) addressing the issue by listing all needed resources per XAML file. Could you please try it out and check if the changes solve your issue?

**Comment by DenZuck** [comment]
Hi,
I'm not able to install the pre-release. 

I get this error:

```javascript
PM> Install-Package MaterialDesignExtensions -Version 2.6.0-a021


Attempting to gather dependency information for package 'MaterialDesignExtensions.2.6.0-a021' with respect to project 'TestMaterialExtensions', targeting '.NETFramework,Version=v4.5'
Install-Package : An error occurred while retrieving package metadata for 'MaterialDesignExtensions.2.6.0-a021' from source 'Syncfusion_WPF'.
  Failed to fetch results from V2 feed at 'https://nuget.syncfusion.com/nuget_wpf/nuget/getsyncfusionpackages/wpf/Packages(Id='MaterialDesignExtensions',Version='2.6.0-a021')' with following message 
: Response status code does not indicate success: 500 (Internal Server Error).
  Response status code does not indicate success: 500 (Internal Server Error).
At line:1 char:1
+ Install-Package MaterialDesignExtensions -Version 2.6.0-a021
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Install-Package], Exception
    + FullyQualifiedErrorId : NuGetCmdletUnhandledException,NuGet.PackageManagement.PowerShellCmdlets.InstallPackageCommand
```

**Comment by spiegelp** [comment]
@DenZuck I can install version 2.6.0-a021 without any problems. Do you use a wrong NuGet feed? The log shows a different feed (nuget.syncfusion.com) than the default NuGet feed.

**Comment by DenZuck** [comment]
The problem was in the Package source. I've installed Synfusion components and new Package sources have been added. Changing from "All" to "nuget.org" I manager to install the pre-release. 

<img width="398" alt="image" src="https://user-images.githubusercontent.com/7376037/58130945-a4113e00-7c1d-11e9-88f4-4d7f1b34e3e4.png">


Using this pre-release there is no more the problem "Cannot find resource named MaterialDesignComboBox".

Thanks

**Comment by spiegelp** [comment]
issue solved by the stable release v2.6.0

**Comment by softwarekerneltechnologies** [comment]
I am getting same error again in MaterialDesignThemes v3.2 at runtime:
System.Windows.Markup.XamlParseException: ''Provide value on 'System.Windows.StaticResourceExtension' threw an exception.' Line number '57' and line position '22'.'
Exception: Cannot find resource named 'MaterialDesignOutlinedTextBox'. Resource names are case sensitive.

System info:
Microsoft Visual Studio Community 2019
Version 16.8.3
